### PR TITLE
Basic backup restore case

### PR DESCRIPTION
**Pull request by soumyalj**

### CODE REVIEW

File: tests/v3_api/test_bkp_restore.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
+import base64
+
+import pytest
+from .common import *  # NOQA
+
+CLUSTER_NAME = os.environ.get("RANCHER_CLUSTER_NAME", "")
```
Comment (sangeethah) [inline tests/v3_api/test_bkp_restore.py]: This is already defined in common.py. Can be removed.

File: tests/v3_api/test_bkp_restore.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        # Create workload after backup
+        testworkload = p_client.create_workload(name=name,
+                                                containers=con,
+                                                namespaceId=ns.id)
+
+        validate_workload(p_client, testworkload, "deployment", ns.name)
+
+        # Perform Restore
+        cluster.restoreFromEtcdBackup(etcdBackupId=backupId)
+        time.sleep(5)
```
Comment (sangeethah) [inline tests/v3_api/test_bkp_restore.py]: Should be able to use validate_cluster() with intermediate_state set to "updating"

File: tests/v3_api/test_bkp_restore.py (hunk 1 above)
Comment (soumyalj) [reply tests/v3_api/test_bkp_restore.py]: removed this env variable

File: tests/v3_api/test_bkp_restore.py (hunk 2 above)
Comment (soumyalj) [reply tests/v3_api/test_bkp_restore.py]: done

**Comment by sangeethah** [comment]
LGTM

**Pull request merged**